core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -573,7 +573,7 @@ public boolean hasLicensesXml() {
                         obsoleteDependencies.add(dependency.getShortName() + "(" + dependency.getVersion() + " < " + d.version + ")");
                     }
                 } else {
-                    disabledDependencies.add(d.toString());
+                    disabledDependencies.add(d.shortName);
```

Comment:
Then in this case I will revert it to the previous version. If the plugin is disabled, it may be obsolete at the same time so I think the required version should be displayed in any case.